pkg/spiffe/spiffe.go

```diff
@@ -52,6 +53,34 @@ var (
 	spiffeLog = log.RegisterScope("spiffe", "SPIFFE library logging", 0)
 )
 
+type Identity struct {
+	TrustDomain    string
+	Namespace      string
+	ServiceAccount string
+}
+
+func ParseIdentity(s string) (Identity, error) {
```

Comment:
Does the client cert authenticator return "spiffe://%s/ns/%s/sa/%s" identity?

Reply:
yep, so does the jwt one

Reply:
Actually not 100% sure about the jwt one - will verify

Reply:
Great,  i am super  sure it is for jwt and kube authn